wapitiCore/net/page.py

```diff
@@ -700,10 +700,16 @@ def html_redirections(self):
         urls = set()
         for meta_tag in self.soup.find_all("meta", attrs={"content": True, "http-equiv": True}):
             if meta_tag and meta_tag["http-equiv"].lower() == "refresh":
-                content_str = meta_tag["content"].lower()
+                content_str = meta_tag["content"]
+                content_str_length = len(meta_tag["content"])
                 url_eq_idx = content_str.find("url=")
```

Comment:
I think we should keep the `.lower()` for the `content_str.find` as "url" may be upper or mixed case.